<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasturClass</class>
 <widget class="QWidget" name="MasturClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mastur</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="header" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="title">
        <property name="text">
         <string>Mastur</string>
        </property>
        <property name="margin">
         <number>2</number>
        </property>
        <property name="indent">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QWidget" name="winBtns" native="true">
        <property name="minimumSize">
         <size>
          <width>136</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>136</width>
          <height>40</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnSettings">
           <property name="minimumSize">
            <size>
             <width>34</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>34</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Mastur.qrc">
             <normaloff>:/assets/img/setting.png</normaloff>:/assets/img/setting.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMini">
           <property name="minimumSize">
            <size>
             <width>34</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>34</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Mastur.qrc">
             <normaloff>:/assets/img/mini.png</normaloff>:/assets/img/mini.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMax">
           <property name="minimumSize">
            <size>
             <width>34</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>34</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Mastur.qrc">
             <normaloff>:/assets/img/max.png</normaloff>:/assets/img/max.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnClose">
           <property name="minimumSize">
            <size>
             <width>34</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>34</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="Mastur.qrc">
             <normaloff>:/assets/img/close.png</normaloff>:/assets/img/close.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="viewWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="home">
      <attribute name="title">
       <string>主页</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="view">
      <attribute name="title">
       <string>浏览</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="TabView" name="tab_view" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="download">
      <attribute name="title">
       <string>下载</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="manage">
      <attribute name="title">
       <string>管理</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="star">
      <attribute name="title">
       <string>收藏</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="settings">
      <attribute name="title">
       <string>设置</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TabView</class>
   <extends>QWidget</extends>
   <header>pages/view/TabView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Mastur.qrc"/>
  <include location="../../Gitlab/EPR-Fitting Qt/fitting.qrc"/>
 </resources>
 <connections/>
</ui>
